### PR TITLE
Add VTTCue settings feature

### PR DESCRIPTION
**Pull request by queengooborg**
This PR creates two new features for the WebVTT API.  These two features were originally suggested as comments in the `webvtt` feature, so this PR does just that!

### CODE REVIEW

**Comment by Elchi3** [comment]
I opened https://github.com/web-platform-dx/web-features/pull/2310 to deal with the constructor

File: features/webvtt-cue-settings.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: WebVTT Cue Settings
+description: The members of the `VTTCue` API can be used to update cues dynamically.
```
Comment (autonome) [inline features/webvtt-cue-settings.yml:2]:
```suggestion
description: The `VTTCue` API updates various aspects of cues dynamically, such as alignment, size and position.
```

File: features/webvtt-cue-settings.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: WebVTT Cue Settings
```
Comment (autonome) [inline features/webvtt-cue-settings.yml:1]:
```suggestion
name: WebVTT cue settings
```

**Review by autonome — CHANGES_REQUESTED** [review]
Suggested couple of fixes, but looks good otherwise w/ these addressed!

**Comment by jamesnw** [comment]
Should this feature also include `css.selectors.cue` and `css.selectors.cue.selector_argument`?

**Comment by Elchi3** [comment]
> Should this feature also include `css.selectors.cue` and `css.selectors.cue.selector_argument`?

Added along with a compute_from.

**Review by autonome — APPROVED** [review]

**Pull request merged**